src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/UpdateSentenceDateService.kt

```diff
@@ -108,39 +110,56 @@ class UpdateSentenceDateService(
     licenceId: Long,
     sentenceChanges: SentenceChanges,
   ) {
-    updatedLicenceEntity.bookingId?.let {
-      prisonApiClient.getHdcStatus(it).defaultIfEmpty(PrisonerHdcStatus(passed = false, approvalStatus = "UNKNOWN"))
```

Comment:
this was a bit of gnarly way to express this.. we know booking id will always be set.

Previously we wouldn't notify about date changes if they had an approved hdc licence - I've changed it so they will continue to be notified if the licence type is HDC.

I think the logic here needs to be reconsidered - there's a potential opportunity to warn the COM that HDC has been approved for an inflight CRD licence, there's also the opportunity to warn that HDC has been refused for an inflight HDC licence. 

This seems like it'll be sufficient for MVP1 but I'll let UCD know. We have a ticket to look at emails in a couple of sprints